plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/DeclarativePipelineMigrationHook.java

```diff
@@ -0,0 +1,50 @@
+package org.jenkins.tools.test.hook;
+
+import org.jenkins.tools.test.model.PomData;
+
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
It looks like these aren't being used. Maybe remove them, either that, or add some log output, similarly to [how it was done for Declarative](https://github.com/jenkinsci/plugin-compat-tester/blob/56bda8384513de71ef501a3f468211def0cf4730/plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/DeclarativePipelineHook.java#L44-L45).

Reply:
similar logging added